code/analytics/src/androidTest/java/com/adobe/marketing/mobile/analytics/internal/AnalyticsIntegrationTests.kt

```diff
@@ -99,6 +99,50 @@ class AnalyticsIntegrationTests {
         assertTrue(countDownLatch.await(1000, TimeUnit.MILLISECONDS))
     }
 
```

Comment:
Move getQueueSize to the top of the test to avoid it to be run before the trackAction/trackState tests for the timing issue.